ts/screens/ErrorScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import { Body, Button, Container, H1, Left, Text, View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import AppHeader from "../components/ui/AppHeader";
+import IconFont from "../components/ui/IconFont";
+import { ReduxProps } from "../store/actions/types";
+
+type OwnProps = {
+  navigation: NavigationScreenProp<NavigationState>;
+};
+type Props = ReduxProps & OwnProps;
+
+const styles = StyleSheet.create({
+  content: {
+    alignItems: "center"
+  },
+  footer: {
+    flexDirection: "row",
+    justifyContent: "space-around"
+  },
+  textCenter: {
+    textAlign: "center"
+  },
+  cancelButton: {
+    backgroundColor: "#556d7a"
+  }
+});
+
+/**
+ * An error screen to show messages when something went wrong
+ */
+class ErrorScreen extends React.Component<Props, never> {
+  public render() {
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button transparent={true}>
+              <IconFont name="io-back" />
+            </Button>
+          </Left>
+          <Body>
+            <Text>Non hai spid?</Text>
```

Comment:
This text (and the other text used in this screen) should be internationalized -- use `I18n`: example [here](https://github.com/teamdigitale/italia-app/blob/91b730310cb32d718a8e5c46b4459b63eec184ef/ts/screens/onboarding/TosScreen.tsx#L97)

Reply:
(missed the WIP bullet about internationalizing it -- 👍 )